_src/pattern-library/sass/components/_grid.scss

```diff
@@ -0,0 +1,1125 @@
+    }
+
+    .post-8 {
+        @include post(8);
+    }
+
+    .post-9 {
+        @include post(9);
+    }
+
+    .post-10 {
+        @include post(10);
+    }
+
+    .post-11 {
+        @include post(11);
+    }
+}
+
+// ----------------------------
+// #CANNED + RESPONSIVE
+// ----------------------------
+.grid-manual {
+
+    // CASE: small screen bp
+    .sm-col-1 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(1); };
+    }
+
+    .sm-col-2 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(2); };
+    }
+
+    .sm-col-3 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(3); };
+    }
+
+    .sm-col-4 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(4); };
+    }
+
+    .sm-col-5 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(5); };
+    }
+
+    .sm-col-6 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(6); };
+    }
+
+    .sm-col-7 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(7); };
+    }
+
+    .sm-col-8 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(8); };
+    }
+
+    .sm-col-9 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(9); };
+    }
+
+    .sm-col-10 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(10); };
+    }
+
+    .sm-col-11 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(11); };
+    }
+
+    .sm-col-12 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include span(full); };
+    }
+
+    // shifts - pre
+    .sm-pre-1 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(1); };
+    }
+
+    .sm-pre-2 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(2); };
+    }
+
+    .sm-pre-3 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(3); };
+    }
+
+    .sm-pre-4 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(4); };
+    }
+
+    .sm-pre-5 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(5); };
+    }
+
+    .sm-pre-6 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(6); };
+    }
+
+    .sm-pre-7 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(7); };
+    }
+
+    .sm-pre-8 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(8); };
+    }
+
+    .sm-pre-9 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(9); };
+    }
+
+    .sm-pre-10 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(10); };
+    }
+
+    .sm-pre-11 {
+        @include susy-breakpoint($bp-screen-sm, $susy-manual) { @include pre(11); };
+    }
+
+    // shifts - post
+    .sm-post-1 {
```

Comment:
NOTE: This whole set of definitions could be made more efficient through a mixin or function, but that seemed like a clean up/optimization task once the grid gets some use.